app/models/course_age_group_monthly_statistics.rb

```diff
@@ -0,0 +1,87 @@
+class CourseAgeGroupMonthlyStatistics
+  def table_data
+    {
+      rows: rows,
+      column_totals: column_totals_for(rows),
+    }
+  end
+
+private
+
+  def rows
+    @rows ||= formatted_group_query.map do |age_group, statuses|
+      {
+        'Age group' => age_group,
```

Comment:
I am wondering whether it's worth keeping the human-readable labels in a .yml file and using snake-cased keys here? That way we could change the labels easily at any time in the future.